### PR TITLE
feat(interactive): Refine `AppBase` and refactor procedure calling, implement call_proc in SDK

### PR DESCRIPTION
**Pull request by zhanglei1949**
- [x] Refine AppBase design and add `InteractiveAppBase`.
- [x] Develop the hqps_proc_handler.
- [x] Implement `call_procedure` method in Java/Python SDKs (submit to query port, not admin port).
- [x] Integrate protoc-java in Java SDK for proto result parsing.
- [x] Incorporate python-plugin in Python SDK for proto result parsing.
- [x] Add unit tests for Java/Python SDKs' procedure calling functionality.

Consider creating a separate `hqps_ic_handler` instance, as using one handler for multiple match_rules in seastar::httpd server may cause segmentation fault upon system exit.

### CODE REVIEW

File: flex/bin/load_plan_and_gen.sh [hunk 1]
```diff
@@ -86,15 +86,20 @@ generate_cpp_yaml() {
   local procedure_name=$1
   local procedure_description=$2
   local output_so_name=$3
-  local rocedure_query=$4
+  local procedure_query=$4
```
Comment (zhanglei1949) [inline flex/bin/load_plan_and_gen.sh:89]: Minor fix.

File: flex/tests/hqps/match_query.h
Comment (zhanglei1949) [inline flex/tests/hqps/match_query.h:1]: This file concludes some codegen apps, should be removed in the future, after runtime execution engine is ready.

File: flex/interactive/sdk/python/interactive_sdk/client/session.py [hunk 2]
```diff
@@ -494,9 +512,32 @@ def get_procedure(
...
+            response = self._query_api.proc_call_with_http_info(
+                graph_id, params_str
+            )
+            if response.status_code == 200:
+                result = CollectiveResults()
+                byte_data = response.data.encode('utf-8')
+                result.ParseFromString(byte_data)
+                return Result.from_response(response)
+            else:
+                return Result.from_response(result)
```
Comment (liulx20) [inline flex/interactive/sdk/python/interactive_sdk/client/session.py:528]: `result` undefined

File: flex/utils/service_utils.cc [hunk 3]
```diff
@@ -17,58 +17,6 @@
 
 namespace gs {
```
Comment (zhanglei1949) [inline flex/utils/service_utils.cc:19]: Those method are inlined to avoid linking flex_utils when doing codegen for cypher queries.

**Review by liulx20 — APPROVED** [review]

**Review by luoxiaojian — APPROVED** [review]

**Review by liulx20 — APPROVED** [review]

**Review by liulx20 — APPROVED** [review]

**Review by lnfjpt — APPROVED** [review]

**Pull request merged**